docs/source/usersguide/install.rst

```diff
@@ -74,6 +74,306 @@ are no longer supported.
 .. _Personal Package Archive: https://launchpad.net/~paulromano/+archive/staging
 .. _APT package manager: https://help.ubuntu.com/community/AptGet/Howto
 
+----------------------------------
+Installing from Source using Spack
+----------------------------------
+
+`Spack`_ is a package management tool designed to support multiple versions and
+configurations of software on a wide variety of platforms and environments.
+Please following Spack's `setup guide`_ to configure the Spack system.
+
+The OpenMC Spack recipe has been configured with variants that match most options
+provided in the CMakeLists.txt file. To see a list of these variants and other
+information:
+
+.. code-block:: sh
+
+    spack info openmc
```

Comment:
I would omit the actual output of this command; otherwise it is going to become out of date every time we do a new release or otherwise change options.

Reply:
Also, it seems better to only mention the `py-openmc` package so that users always get the Python API.

Reply:
yup, agreed